include/constants/move_effects.h

```diff
@@ -0,0 +1,276 @@
+#ifndef POKEHEARTGOLD_CONSTANTS_MOVE_EFFECTS_H
+#define POKEHEARTGOLD_CONSTANTS_MOVE_EFFECTS_H
+
```

Comment:
I think a lot of move effects could be more concise in their names, also might need clarity whether it affects self, opponant, or (potentially) allies

Reply:
Move effect is ALWAYS based on Targeting, so self-targeting and ally-targeting get handled by separate variables